### ISSUE TITLE
Runtime error: espeak not installed on your system

### ISSUE THREAD

**Issue by davidkamp** [haoheliu/AudioLDM2#19]
I am on a M1 macbook air on Monterey and followed the latest install instructions.
when I run the audiodlm2 example command "Musical constellations ...etc" I get this runtime error.

When I try "pip3 install espeak" no matching distribution is found, if I run "pip3 install phonemizer" it says it is already installed....
Any ideas?

**Comment by robertZaufall** [comment]
`brew install espeak` installs this but does not clear the error

**Comment by davidkamp** [comment]
yes, doing that tells me espeak  is installed, same runtime error though.

**Comment by haoheliu** [comment]
Oh yes I forgot this dependency. You can do the following

```
sudo apt-get install espeak
```

**Comment by davidkamp** [comment]
espeak is listed as installed when using `brew install espeak ` (I believe apt-get does not exist on osx?)

**Comment by robertZaufall** [comment]
this solved it for me:
`brew install espeak`
`export PHONEMIZER_ESPEAK_LIBRARY=/opt/homebrew/Cellar/espeak/1.48.04_1/lib/libespeak.1.1.48.dylib`

**Comment by robertZaufall** [comment]
Version idependent:

`export PHONEMIZER_ESPEAK_LIBRARY=$(find /opt/homebrew/Cellar/espeak -name "libespeak.dylib" | sort -r | head -n 1)`

Also consider to add this line to your `.zshenv` file for persistence.

**Comment by marianbastiUNRN** [comment]
setting PHONEMIZER_ESPEAK_LIBRARY environment variable worked for me also on Windows 10

**Comment by davidkamp** [comment]
Thank you  - this worked for me too!
